src/workloads/scale/MixedWorkloadsGennyRateLimited.yml

```diff
@@ -0,0 +1,197 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload is a port of the mixed_workloads in the workloads
```

Comment:
Please update description to reference the other workload and call out that this one is rate limited.